vyper/codegen/stmt.py

```diff
@@ -142,7 +142,8 @@ def _get_last(ir):
         # TODO maybe use ensure_in_memory
         if msg_ir.location != MEMORY:
             buf = self.context.new_internal_variable(msg_ir.typ)
-            instantiate_msg = make_byte_array_copier(buf, msg_ir)
+            dst = IRnode.from_list(buf, typ=msg_ir.typ, location=MEMORY)
```

Comment:
magic!

Reply:
does `ensure_in_memory` work here btw?

Reply:
I tried doing this but the same error is thrown:
```
if msg_ir.location != MEMORY:
    buf = self.context.new_internal_variable(msg_ir.typ)
    instantiate_msg = ensure_in_memory(msg_ir, self.context)
else:
    ...
```

Reply:
uhm, i think if we use ensure_in_memory we don't need to branch